sierra_adapter/sierra_reader/src/main/scala/uk/ac/wellcome/platform/sierra_reader/services/SierraReaderWorkerService.scala

```diff
@@ -37,16 +37,16 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class SierraReaderWorkerService(
   sqsStream: SQSStream[NotificationMessage],
-  s3client: AmazonS3,
   s3Config: S3Config,
```

Comment:
Why do we have to pass around a client and a config? Can we not build one from the other?

Reply:
The config is telling us which bucket to use in S3, which is different from how actually connect to S3, although I can see it might not be immediately obvious.

Reply:
Ah, is the config that the client is built from different to the one that gets passed in?

Reply:
Different to the one that gets passed in here, yep.

(I don't think we use any config directly for the client, actually, we just pull in creds from the environment.)